src/LEGO.AsyncAPI/Models/AsyncApiConstants.cs

```diff
@@ -138,5 +138,10 @@ public static class AsyncApiConstants
         public const string MaxMessageBytes = "max.message.bytes";
         public const string TopicConfiguration = "topicConfiguration";
         public const string GeoReplication = "geo-replication";
+        public const string Policy = "policy";
```

Comment:
I’ll change it to hard coded strings. Just curious though because other bindings are still using these. What was the intention in introducing this constants file to map to property names?

Reply:
It's a leftover from the old bindings. Which were in the same project as the constants file.
But they should be moved out - or a new constants file created.